DEVELOPING_A_RUNTIME.md

```diff
@@ -267,16 +267,18 @@ This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaSc
 ### `Lambda`
 
 ```ts
-import { Lambda } from '@now/build-utils'
+import { Lambda } from '@now/build-utils';
 ```
 
-This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), called a Serverless Function, that can be created by supplying `files`, `handler`, `runtime`, and `environment` as an object to the [`createLambda`](#createlambda) helper. The instances of this class should not be created directly. Instead use a call to [`createLambda`](#createlambda).
+This is a [JavaScript class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), called a Serverless Function, that can be created by supplying `files`, `handler`, `runtime`, and `environment` as an object to the [`createLambda`](#createlambda) helper, the properties `memory` and `maxDuration` are optional. The instances of this class should not be created directly. Instead use a call to [`createLambda`](#createlambda).
 
 **Properties:**
 
 - `files : Files` the internal filesystem of the lambda
 - `handler : String` path to handler file and (optionally) a function name it exports
 - `runtime : LambdaRuntime` the name of the lambda runtime
+- `memory : Number` the memory size in MB the lambda should use, a value from 128 to 3008, in steps of 64
+- `maxDuration: Number` seconds after this lambda will time out
```

Comment:
```suggestion
- `maxDuration: Number` number of seconds that the lambda is allowed to run before timing out
```

Reply:
As a more general comment for this "properties" section: should we migrate from "lambda" to "serverless function" nomenclature?

Reply:
@TooTallNate Absolutely! The only reason why we kept "Lambda" here is because of the types. Once they're migrated, we'll also transition the rest of the text.

The documentation already uses "serverless functions", by the way. Great catch!